packages/adapter-node/src/index.js

```diff
@@ -83,7 +90,7 @@ server.server.on(
 				server.server.closeIdleConnections();
```

Comment:
hmm. I wonder if users would want an event to know that all requests have finished processing. if you issue a `sveltekit:shutdown` when receiving `SIGINT` and shutdown your database connections then maybe your requests can't actually finish and fail because the database connection is gone, which would kind of defeat the purpose of having a shutdown timeout.

maybe it'd be better to issue an event here rather than in `shutdown`. if users want to do something earlier they can still listen for `SIGTERM` or `SIGINT` individually without us firing an event that simply echoes them (https://github.com/sveltejs/kit/pull/12153/files#r1637507349)

Reply:
That was the idea. The `sveltekit:shutdown` event is emitted inside the `server.close((error) => { ... }); callback. And that callback runs after all requests have finished and the server closed.